### DISCUSSION TITLE
Something like Hires. fix

### DISCUSSION THREAD

**Discussion by Bullseye110** [lllyasviel/Fooocus#872] [Ideas]
Hello!
So I used Fooocus for some time now and it is actually kinda the best and the easiest UI at this moment.
But I think that it needs something like hires. fix to be complete. Or something else which can upscale the image while generating it/or immediately after generating it. You know what I'm saying.
It'll be good if it will have the possibility of using UltraSharp, AnimeSharp or NMKD upscalers.
Thanks and keep up the good work!

And yea, it works fine in google colab now.

**Comment by williamsblood** [comment]
Quote me on this one, I would love to have a automatic upscale 1.5 or 2.0 after every creation,
